OutOfSchool/Directory.Packages.props

```diff
@@ -61,7 +61,7 @@
     <PackageVersion Include="OpenIddict.Client.SystemNetHttp" Version="$(OpenIdDictVersion)" />
     <PackageVersion Include="OpenIddict.EntityFrameworkCore" Version="$(OpenIdDictVersion)" />
     <PackageVersion Include="OpenIddict.Quartz" Version="$(OpenIdDictVersion)" />
-    <PackageVersion Include="OpenIddict.Validation.AspNetCore" Version="$(OpenIdDictVersion)" />
+    <PackageVersion Include="OpenIddict.Validation.AspNetCore" Version="6.1.1" />
```

Comment:
it is better to have unified version. So return the `"$(OpenIdDictVersion)"` variable usage here and change the variable value at the beginning of the file.